amp.js

```diff
@@ -80,6 +80,7 @@ createTask('test-report-upload', 'testReportUpload', 'test-report-upload');
 createTask('unit', 'unit', 'unit');
 createTask('update-packages', 'updatePackages', 'update-packages');
 createTask('validator', 'validator', 'validator');
+createTask('validator-cpp-test', 'validatorCppTest', 'validator');
```

Comment:
usability / consistency nit: Almost all these task names are tests, so I think we should drop the `test` suffix and call this just `validator-cpp`.

Reply:
Done